webapp/src/codecard.tsx

```diff
@@ -122,8 +126,10 @@ export class CodeCardView extends data.Component<pxt.CodeCard, CodeCardState> {
                     </div>
                 </div> : undefined}
             {(card.shortName || name || descriptions) ?
-                <div className="content">
-                    {card.shortName || name ? <div className="header">{card.shortName || name}</div> : null}
+                <div className={`content ${this.props.tallCard? "tall" : ""}`}>
+                    {card.shortName || name ? <div className="header">{card.shortName || name}
+                            <div className="tags">{card.tags}</div>
```

Comment:
I'm not certain, but do we want an rlf here? I can't recall if these can / should be translated so deferring to richard / shannon

Also, should the tags have a borderbox similar to the display on the skillmap
![image](https://user-images.githubusercontent.com/5615930/133828751-2dd28adc-b7a4-4954-9ad9-08b62623a419.png)
 or maybe just separate with commas? Might be just me but the spacing looks close enough that I'd read it as one phrase rather than separate tags.

Reply:
oooh yeah I wasn't thinking about translations

and for the tags I copied the design mentioned in office hours -- i'm not sure if there'll be more than one tag (these were just for test). I think we can just add commas in authoring? but we can take to office hours again!

Reply:
Ah sounds good then, I think I missed that office hours so my bad~